json_to_csv.py

```diff
@@ -79,7 +80,22 @@ def reduce_item(key, value):
 
         processed_data = []
         header = []
+
+        acceptable_keys = []
+        #reading the field that needs to be saved
+        if len(sys.argv) > 4:
+            for i in range(4,len(sys.argv)):
+                acceptable_keys.append(sys.argv[i])
+            print('Only copying from: ',end='')
```

Comment:
This line is throwing syntax error.

Reply:
Answering your questions
1. The acceptable fields are only at level one. The arguments specify which fields to keep in the final output csv file.
2. Fields not mentioned in the arguments are excluded from the final output csv file.
3. Right now only level one fields are being worked upon, I can write something to handle complex relationships. For `parent2` argument could be `parent1.parent2`
4-5. Can you expand on that, I don't know which list or dictionary you are talking about. Also can you give a example of homogeneous object.
For the syntax error can you share the error message? `end = ''` might be the culprit but I cannot be sure.
Also I could also add a delete feature. So with arguments starting with `-field1 field2` these fields would be removed from the final output csv file.